docs/Bundler/api/get-useroperation-receipt.md

```diff
@@ -35,110 +36,387 @@ Body
 
 ## Response
 
+`null` in case the UserOperation is not yet included in a block, or:
+
+- userOpHash the request hash
+- entryPoint
+- sender
+- nonce
+- paymaster the paymaster used for this userOp (or empty)
+- actualGasCost - actual amount paid (by account or paymaster) for this UserOperation
+- actualGasUsed - total gas used by this UserOperation (including preVerification, creation, validation and execution)
+- success string - did this execution completed without revert
+- reason in case of revert, this is the revert reason
+- logs the logs generated by this UserOperation (not including logs of other UserOperations in the same bundle)
+- receipt the TransactionReceipt object. Note that the returned TransactionReceipt is for the entire bundle, not only for this UserOperation.
+
+Example response for Polygon:
```

Comment:
can we convey users about the differences between the two receipts? and mention how it is different for L1/ L2 and then mention an example from polygon for L2. 
also this json can be embedded in a details tag.